src/Couchbase.TravelSample.Tests/AirlineTests.cs

```diff
@@ -0,0 +1,346 @@
+using System.Net;
+using System.Text;
+using Couchbase.TravelSample.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Newtonsoft.Json;
+
+namespace Couchbase.TravelSample.Tests;
+
+public class AirlineTests : IClassFixture<WebApplicationFactory<Program>>
+{
+    private readonly HttpClient _client;
+    private const string BaseHostname = "/api/v1/airline";
+
+    public AirlineTests(WebApplicationFactory<Program> factory)
+    {
+        _client = factory.CreateClient();
+    }
+
+    [Fact]
+    public async Task TestListAirlinesInCountryWithPaginationAsync()
+    {
+        // Define parameters
+        const string country = "United States";
+        const int pageSize = 3;
+        const int iterations = 3;
+        var airlinesList = new HashSet<string>();
```

Comment:
Is there a very specific reason you need HashSet and are using a for loop for iterations vs a list or an array?  You already know how many records are going to be in this so what is the justification for using a HashSet?